shared/app/push-prompt.native.tsx

```diff
@@ -13,73 +15,110 @@ type Props = {
 }
 
 const PushPrompt = (props: Props) => (
-  <Kb.ScrollView style={styles.scroll} contentContainerStyle={styles.scrollContent}>
-    <Kb.Box2 direction="vertical" fullHeight={true} fullWidth={true} gap="small" style={styles.container}>
-      <Kb.Box style={styles.spacer} />
-      <Kb.Text center={true} type="Header" style={styles.text}>
-        Please turn on notifications!
-      </Kb.Text>
-      <Kb.RequireImage
-        resizeMode="contain"
-        style={styles.image}
-        src={require('../images/illustrations/illustration-turn-on-notifications-460-x-252.png')}
-      />
-      <Kb.Text center={true} type="BodySmallSemibold" style={styles.text}>
-        It's{' '}
-        <Kb.Text center={true} type="BodySmallSemiboldItalic" style={styles.text}>
-          very
-        </Kb.Text>{' '}
-        important you enable notifications.
-      </Kb.Text>
-      <Kb.Text center={true} type="BodySmall" style={styles.text}>
-        This phone may need to perform crypto for you, which the Keybase servers cannot do. For example, if
-        you provision a new device, this phone will be contacted.
-      </Kb.Text>
+  <Kb.Box2 direction="vertical" fullHeight={true} fullWidth={true} gap="small" style={styles.container}>
+    <Kb.RequireImage
+      resizeMode="stretch"
+      style={styles.image}
+      src={require('../images/illustrations/illustration-turn-on-notifications-460-x-252.png')}
+    />
+    <Kb.Text center={true} type="BodySmallSemibold" style={styles.text}>
+      It's{' '}
+      <Kb.Text center={true} type="BodySmallSemiboldItalic" style={styles.text}>
+        very
+      </Kb.Text>{' '}
+      important you enable notifications.
```

Comment:
The copy is tweaked in the designs

Reply:
Yeah this was as of last night. I'll update it.